fax/implicit/twophase.py

```diff
@@ -2,6 +2,7 @@
 import logging
 
 import jax
+from jax import numpy as np, tree_util
```

Comment:
Could you split this into two lines?

Reply:
Yup, good catch.  I think isort prefers that.